mcapi/crypto.c

```diff
@@ -424,7 +424,7 @@ int CRYPT_TDES_CBC_Decrypt(CRYPT_TDES_CTX* tdes, unsigned char* out,
 int CRYPT_AES_KeySet(CRYPT_AES_CTX* aes, const unsigned char* key,
                      unsigned int keyLen, const unsigned char* iv, int dir)
 {
-    typedef char aes_test[sizeof(CRYPT_AES_CTX) >= sizeof(Aes) ? 1 : -1];
+    typedef char aes_test[sizeof(CRYPT_AES_CTX) >= sizeof(Aes) ? 1 : 0];
```

Comment:
The `[-1]` here is deliberate, to derail compilation.  the code pattern is quite common:
```
$ egrep -c '\[[^]]*:[[:space:]]*-1[[:space:]]*\]' mcapi/*.c src/*.c | fgrep -v :0
mcapi/crypto.c:9
src/internal.c:7
src/sniffer.c:1
src/ssl.c:14
src/tls13.c:6
```